packages/sdk/src/web/experimental/index.ts

```
@@ -91,22 +89,21 @@ export async function init<T extends SchemaType>(options: SDKConfig) {
+                      (await client.getEventMessages(query)).items
                   )
                 : parseEntities<T>(
-                      await client.getEventMessages(query, historical)
+                      (await client.getEventMessages(query)).items
                   );
             return [
                 events,
                 client.onEventMessageUpdated(
                     intoEntityKeysClause<T>(query.clause, events),
-                    (entityId: string, entityData: any) => {
+                    (_: string, entityData: any) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)